.ci/index.ts

```diff
@@ -100,9 +100,9 @@ export = async function main() {
     actions: {
       alwaysOnline: 'on',
       cacheLevel: 'cache_everything',
-      cacheTtlByStatuses: [{ codes: '200', ttl: 3600 /* an hour */ }],
-      edgeCacheTtl: 3600 /* an hour */,
-      browserCacheTtl: '3600' /* an hour */,
+      cacheTtlByStatuses: [{ codes: '200', ttl: 31536000 /* a year */ }],
```

Comment:
ain't there `31557600` seconds in a year?

Reply:
<img width="841" alt="image" src="https://user-images.githubusercontent.com/24811313/177616181-52b499c5-5e4f-462d-bdef-99cf18005581.png">

didn't do the math, just googled it